web/frontend/src/pages/form/components/SubjectComponent.tsx

```diff
@@ -46,15 +46,15 @@ const SubjectComponent: FC<SubjectComponentProps> = ({
   const isSubjectMounted = useRef<boolean>(false);
   const [isOpen, setIsOpen] = useState<boolean>(false);
   const [titleChanging, setTitleChanging] = useState<boolean>(
-    subjectObject.Title.length ? false : true
+    subjectObject.Title.En.length ? false : true
```

Comment:
Does this assure that we always have an English title?

Reply:
no, the schema defined in `web/frontend/src/schema/configurationValidation.ts` is

which is called when trying to submit the form

Reply:
Is this also called when submitting via JSON?

Reply:
yes, the schema validation is also called when uploading a JSON